regimes/it/examples/freelance.json

```diff
@@ -80,7 +80,7 @@
         {
           "cat": "IRPEF",
           "percent": "20.0%",
-          "rate": "self-employed-habitual"
+          "code": "A"
```

Comment:
Thinking about it some more, I'm not suer-happy with this approach. The fundamental problem here is that there is no reference to what the (in this case) `"A"` comes from or implies. Perhaps a better approach would be to add the concept of a "codes hash", so that the usage could be more like:

```json
{
  "cat": "IRPEF",
  "percent": "20.0%",
  "codes": {
    "it-sdi-nature": "A"
  }
}
```

Reply:
Agreed the second example is better that the original change. And "codes" could become "ext" as proposed [here](https://github.com/invopop/gobl/pull/183#issuecomment-1683528947).